### PR TITLE
Feat: Added Grid and Slider

### PR DESCRIPTION
**Pull request by sainath7878**
Added Grid and Slider Components
Grid
Screenshot 2022-02-21 at 4 20 10 AM
Screenshot 2022-02-21 at 4 20 31 AM

Slider
Screenshot 2022-02-21 at 4 21 09 AM

Screenshot 2022-02-21 at 4 21 24 AM

### CODE REVIEW

File: component/slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.slider-enabled > input{
+    appearance: none;
+    background-color: var(--SECONDARY-COLOR);
+    border-radius: 10px;
+    width: 200px;
```
Comment (05him) [inline component/slider/slider.css]: suggestion :- try avoiding px in the width, height and such things, instead of them you can use rem,em,%

**Review by 05him — APPROVED** [review]
The code is good
just one minute suggestion which you can go through
else its Good To Go

edit:- check your netlify link is it working or not, as its not showing anything

**Review by aryanandarnav13 — APPROVED** [review]
Looks Good

File: component/slider/slider.css (hunk 1 above)
Comment (sainath7878) [reply component/slider/slider.css]: Done.

**Comment by sainath7878** [comment]
> The code is good just one minute suggestion which you can go through else its Good To Go
> 
> edit:- check your netlify link is it working or not, as its not showing anything

The link contains only css so, nothing would be visible.

**Pull request merged**